### DISCUSSION TITLE
hope can notified to  app  “bark” notifications

### DISCUSSION THREAD

**Discussion by guojun-max** [containrrr/watchtower#1278] [Feature Requests]
hope can notified to  app  “bark” notifications

time="2022-04-29T16:12:37+08:00" level=info msg="Using notifications: smtp"
 support  smtp,   hope  support  “bark” message notifications

such as send to https://api.day.app/apikey/message

**Comment by piksel** [comment]
Implemented in https://github.com/containrrr/shoutrrr/pull/234

**Reply by guojun-max** [reply]
how to setup this feature
